src/components/KTruncate/KTruncate.vue

```diff
@@ -282,30 +287,35 @@ onUnmounted(() => {
   .collapse-trigger {
     background-color: var(--kui-color-background-primary-weakest, $kui-color-background-primary-weakest);
     border-radius: var(--kui-border-radius-round, $kui-border-radius-round);
+    color: var(--kui-color-text-primary, $kui-color-text-primary);
     cursor: pointer;
+    outline: none;
+    padding: var(--kui-space-20, $kui-space-20);
 
     &:focus-visible {
       box-shadow: var(--kui-shadow-focus, $kui-shadow-focus);
     }
 
     &:hover {
       background-color: var(--kui-color-background-primary-weaker, $kui-color-background-primary-weaker);
+      color: var(--kui-color-text-primary-strong, $kui-color-text-primary-strong) !important;
     }
 
-    &:focus {
+    &:focus, &:focus-within {
```

Comment:
nit: drop to new line
```suggestion
    &:focus, 
    &:focus-within {
```